server/src/BitBakeDocScanner.ts

```diff
@@ -43,9 +44,12 @@ const variableInfosOverrides: Record<string, VariableInfosOverride> = {
 const variablesFolder = 'doc/bitbake-user-manual/bitbake-user-manual-ref-variables.rst'
 const variablesRegexForDoc = /^ {3}:term:`(?<name>[A-Z_]*?)`\n(?<definition>.*?)(?=^ {3}:term:|$(?!\n))/gsm
 
+const yoctoTaskFilePath = path.join(__dirname, '../src/yocto-docs/tasks.rst')
```

Comment:
I suggest adding an npm script to update this file. We should do the same for the variables hover definitions from @idillon-sfl . For integration tests, we'll also need to fetch a full poky. Suggestion:

1. Add a `npm run fetch-poky` script which downloads and extracts a set poky archive
2. Add a `npm run extract-poky-docs` script which copies the files required in our sources to proper locations
3. Move imported yocto files in a "ressources" directory

I'll start implementing parts of this for integration tests.

Reply:
Don't fetch poky, but https://git.yoctoproject.org/yocto-docs/. That's the canonical source of the documentation.